portfolio/src/main/webapp/style.css

```
@@ -252,6 +252,14 @@ h3 {
 
 /* comments.html */
 
+#comments-header {
+  margin-top:20px;
+}
+
+#login-out, #set-nickname-button {
```

Each id should be on a new line.
```suggestion
#login-out,
#set-nickname-button {
```

---

ok